src/popup.tsx

```diff
@@ -56,6 +56,8 @@ function Popup() {
     browser.tabs.create({
       url: browser.runtime.getURL('options.html'),
       active: true
+    }).then(() => {
+      window.close();
```

Comment:
Why is this here?

Reply:
Without this, we have to manually go back after pressing the Options button to see `options.html`.

<details><summary>Screenshot</summary>

![20230304_050200](https://user-images.githubusercontent.com/65112898/222815777-15175f89-ae85-43ef-8d45-315c258aa721.png)

</details>

Reply:
Got it :+1: